lib/fuelsdk/soap.rb

```diff
@@ -27,7 +27,7 @@ def unpack_body raw
       def unpack raw
         @code = raw.http.code
         unpack_body raw
-        @success = @message == 'OK'
+        @success = raw.hash[:envelope][:body][:create_response][:results][:status_message] == "Created TriggeredSend"
```

Comment:
Is this SOAP request class a generic utility used by ET elsewhere - for example when we check if emails are bounced? If so, this change will break that code. See https://github.com/simplybusiness/Skynet/blob/master/lib/skynet/exact_target/bounced_emails_presenter.rb for example.

Reply:
Shouldn't condition be a conjunction, ie `@message == 'OK' && status_message == "CreatedTriggeredSend"` ?

Reply:
Doh! I didn't think of that.